docs/content/docs/ops/state/checkpoints_vs_savepoints.md

```diff
@@ -0,0 +1,97 @@
+i) being as lightweight to create and ii) being as fast to restore from as possible. 
+Optimizations towards those goals can exploit certain properties, e.g. that the job code doesn't change between the execution attempts. 
+Checkpoints are usually dropped after the job was terminated by the user (except if explicitly configured as retained Checkpoints).
+
+{{< hint info >}}
+- Checkpoints are automatically deleted if the application is terminated by the user 
+(except if checkpoints are explicitly configured to be retained).
+- Checkpoints are stored in state backend-specific (native) data format (may be incremental depending on the specific backend).
+{{< /hint >}}
+
+Although [savepoints]({{< ref "docs/ops/state/savepoints" >}}) are created internally with the same mechanisms as
+checkpoints, they are conceptually different and can be a bit more expensive to produce and restore from and focus
+more on portability and flexibility with respect to changes to the job. 
+Their use case is planned, manual backup and resume. For example, this could be an update of your Flink version, changing your job graph,
+changing parallelism, forking a second job like for a red/blue deployment, and so on.
+
+{{< hint info >}}
+- Savepoints are created, owned and deleted solely by the user. 
+That means, Flink does not delete savepoints neither after job termination nor after
+restore.
+- Savepoints are stored in a state backend independent (canonical) format (Note: Since Flink 1.15, savepoints can be also stored in
+the backend-specific [native]({{< ref "docs/ops/state/savepoints" >}}#savepoint-format) format which is faster to create
+and restore but comes with some limitations.
+{{< /hint >}}
+
+### Capabilities and limitations
+The following table gives an overview of capabilities and limitations for the various types of savepoints and
+checkpoints.
+- ✓ - Flink fully support this type of the snapshot
+- x - Flink doesn't support this type of the snapshot
+- ! - in fact, Flink support this type but officially, Flink doesn't support this type of the snapshot(there is certain level of risk to use it)
+
+| Operation                       | Canonical Savepoint | Native Savepoint | Aligned Checkpoint | Unaligned Checkpoint |
+|:--------------------------------|:--------------------|:-----------------|:-------------------|:---------------------|
+| State backend change            | ✓                   | x                | x                  | x                    |
+| State Processor API(writing)    | ✓                   | x                | x                  | x                    |
+| State Processor API(reading)    | ✓                   | !                | !                  | x                    |
+| Self-contained and relocatable  | ✓                   | ✓                | x                  | x                    |
+| Schema evolution                | ✓                   | !                | !                  | !                    |
```

Comment:
Are the values for `Schema evolution` correct?

Reply:
Haven't we said we want to either update it, or remove it from the current version? @pnowojski

Reply:
I don't remember. Have we?